chrome-extension/content.css

```diff
@@ -1,16 +1,17 @@
-.injection-sidebar {
+#extension-sidebar {
   z-index: 9999;
   position: fixed;
-  right: 0;
-  top: 20px;
+  right: 0px;
+  top: 0px;
```

Comment:
No units for 0 https://google.github.io/styleguide/htmlcssguide.html#0_and_Units